.github/workflows/build.yml

```
@@ -195,9 +195,70 @@ jobs:
           tar cfz "${{ env.RELEASE_NAME }}.tgz" "${{ env.RELEASE_NAME }}/"
 
       - name: Upload to Release
-        if: env.RELEASE == 'true'
+        if: env.RELEASE == 'true' && !contains(matrix.artifact_name, 'ios')

```

すみません、`voicevox_core` の方の変更に合わせて iOS の場合で各バイナリごとにアップロードしないようにされたのだと思うのですが、このリポジトリの場合は [`onnxruntime-rs` のビルド時にダウンロード対象となる](https://github.com/VOICEVOX/onnxruntime-rs/blob/ebb9dcb9b26ee681889b52b6db3b4f642b04a250/onnxruntime-sys/build.rs#L26-L27)ため、こちらではアップロードするようお願いします 🙇‍♂️

```suggestion
        if: env.RELEASE == 'true'
```

---

おっしゃるとおりですね。
自分で作っておいて失念しておりました。ご指摘ありがとうございます。
d3ab74d で対応しました。